Update motivation (#23)
tc39/proposal-await.ops: README.md

> # await operation proposal

Reviewer: ```suggestion
# await operations proposal
```

The [rendered spec text](https://tc39.es/proposal-await.ops/). [Playground Link](https://www.staging-typescript.org/play?ts=4.0.0-pr-39224-4#code/IYZwngdgxgBAZgV2gFwJYHsIwB4AoCUMA3gFAzkzADuwqyAdMADZMwAMZF1tDATsFACmMANoAFXugC2qEIPq9BIdEwBug3AEZ8AXU7ludRiwDKg5MiaCAJqL0BfIA)

Introduce await.all / await.race / await.allSettled / await.any to simplify the usage of async functions

Stage: 1

Champions: [Jack Works](https://github.com/Jack-Works), [Jordan Harband](https://github.com/ljharb)

## Motivation

> When developers use async functions, they have to know about Promise if they want to handle multiple tasks parallelly (`Promise.all` stuff), this is some kind of "abstraction leak".

Reviewer: ```suggestion
When developers use async functions, they have to know about Promise if they want to handle multiple tasks concurrently (`Promise.all`, eg), this is some kind of "abstraction leak".
```

> This proposal is intended to fix the problem by introducing parallel Promise utils in the async-await style.

Reviewer: ```suggestion
This proposal is intended to fix the problem by introducing concurrent Promise utils in the async-await style.
```

await.any expr
// eq: await Promise.any(expr)
```

## Alternative solution

Introduce a new global `Async` (or `Await`), then let

- `Async.all` (equals to `Promise.all`)
- `Async.race` (equals to `Promise.race`)
- `Async.allSettled` (equals to `Promise.allSettled`)

> - `Async.any` (equals to `Promise.any`)

Reviewer: I’m very confused by this - adding a new global isn’t going to address the confusion caused by the ability to use syntax for await but not for the combinators.

Author: oh yeah, I found this way we need `await Async.all(...)` which is, ... a bit strange. let me remove this part.
